apps/demo-fuels/fuels.config.ts

```diff
@@ -0,0 +1,8 @@
+import { createConfig } from 'fuels';
+
+export default createConfig({
+  workspace: './sway-programs',
```

Comment:
Will the CLI also work in case someone is not using TypeScript? i.e. will it also detect `fuels.config.js`?

Reply:
Yes, now it does [accept](https://github.com/FuelLabs/fuels-ts/pull/1291/commits/1d02233ceb60b16ff852d2844101f928e22c10df) `.js`, and [also](https://github.com/FuelLabs/fuels-ts/pull/1291/commits/62ac3fa15058a244246915710ecd256949e7d3e5) `.cjs` and `.mjs`.